spec/api/v1/users_controller_spec.rb

```diff
@@ -0,0 +1,49 @@
+require 'rails_helper'
+include Warden::Test::Helpers
+
+RSpec.describe Api::V1::UsersController, type: :controller do
+
+
+  describe "GET#index" do
+    let!(:user) {FactoryBot.create(:user, id: 99)}
+    let!(:city_one) {City.create(city_name: "Boston", state: "MA", user_id: user.id)}
```

Comment:
I would use `create!` in these `let` invocations because we are expecting these records to be written to the database.